pkg/cloud/services/compute/instance.go

```diff
@@ -182,6 +184,12 @@ func createInstance(is *Service, clusterName string, i *infrav1.Instance) (*infr
 			port = portList[0]
 		}
 
+		for _, fip := range port.FixedIPs {
```

Comment:
Just to confirm some assumptions:
* if AccessSubnet is set we will get a hit here and have a value for `accessIPv4`
* otherwise accessIPv4 and we have no change to before this PR

Reply:
yes, exactly

Reply:
Okay, thx :)